### DISCUSSION TITLE
[SOLVED] fresh install: can't access AzuraCast

### DISCUSSION THREAD

**Discussion by curchunflo** [AzuraCast/AzuraCast#4449] [General]
Hello, how are you? At first: thank you so much for your work.
I am part of the RadioPedal.uy collective, a self-managed non profit digital radio from Montevideo, Uruguay. We are looking for software that allows us to automate the radio and operate live, always on Linux, since we are in favor of free software (free as freedom).
We installed AzuraCast and configured it, the installer threw some errors and then continued, normally. When entering localhost from my browser, the apache page appears in the place of AzuraCast, could you guide me on how to finish installing it? I am shure that is a configuration problem, but I cant figure it out wich kind of problem or where to modify. I did not have a notification and I do not know where to start to fix it. Sorry if I am not giving you all the information you need, but if you askme for any kind of information (like port fowarding or all kind stuff you need to get a diagnose), I'll be glad to give to you.
Thank you in advance! Really, thank you very much!

**Comment by BusterNeece** [comment]
@curchunflo How did you install AzuraCast? Using the Docker or the Ansible installation method?

It's possible that Apache is also installed on your server and you need to disable or completely uninstall it for our own web server software (nginx) to run instead. That may be as simple as running `apt-get remove apache2`, then restarting the server.

**Reply by curchunflo** [reply]
hi @SlvrEagle23 thank you very much for your answer, now I've uninstalled apache2 and have a new problem, when I go to localhost, I only see nginix homepage, the install's log has some errors, I paste the output here, maybe it helps to know where is the problem. I've run twice the install script, the first one it didnt detect my public ip, the second, it did it, but I want to use it with localhost due I can't configure a static IP in my network. 
I can't find where to modify the configuration (there is any file I can edit after installation without get in localhost?)
I am running linux mint

root@xxx:/var/azuracast# ./docker.sh install
Docker is already installed! Continuing...
Docker Compose is already installed and up to date! Continuing...
curl: (22) The requested URL returned error: 404 
ERROR: .FileNotFoundError: [Errno 2] No such file or directory: './docker-compose.installer.yml'
ERROR: .FileNotFoundError: [Errno 2] No such file or directory: './docker-compose.installer.yml'
rm: no se puede borrar 'docker-compose.installer.yml': No existe el archivo o el directorio
Pulling mariadb                 ... done
Pulling redis                   ... done
Pulling stations                ... done
Pulling web                     ... done
Pulling nginx_proxy             ... done
Pulling nginx_proxy_letsencrypt ... done
Creating network "azuracast_backend" with driver "bridge"
Creating network "azuracast_frontend" with driver "bridge"
Creating volume "azuracast_nginx_proxy_vhosts" with default driver
Creating volume "azuracast_db_data" with default driver
Creating volume "azuracast_letsencrypt" with default driver
Creating volume "azuracast_letsencrypt_html" with default driver
Creating volume "azuracast_letsencrypt_acme" with default driver
Creating volume "azuracast_shoutcast2_install" with default driver
Creating volume "azuracast_geolite_install" with default driver
Creating volume "azuracast_sftpgo_data" with default driver
Creating volume "azuracast_station_data" with default driver
Creating volume "azuracast_www_vendor" with default driver
Creating volume "azuracast_tmp_data" with default driver
Creating volume "azuracast_redis_data" with default driver
Creating volume "azuracast_backups" with default driver
Creating azuracast_redis    ... done
Creating azuracast_stations ... done
Creating azuracast_mariadb  ... done
Creating azuracast_web_run  ... done
Starting up AzuraCast services...
Services started up and ready!
AzuraCast Setup
(Environment: production)

Configurar AzuraCast
====================

Bienvenido a AzuraCast. Por favor, espere mientras se configuran algunas dependencias clave de AzuraCast...
 * Entorno: Production
 * Método de instalación: Docker


Ejecutando Migraciones de Base de Datos
---------------------------------------



Generando Clases de Proxy de Base de Datos
------------------------------------------


 Processing entity "App\Entity\Settings"
 Processing entity "App\Entity\SongHistory"
 Processing entity "App\Entity\Analytics"
 Processing entity "App\Entity\StorageLocation"
 Processing entity "App\Entity\StationMedia"
 Processing entity "App\Entity\Station"
 Processing entity "App\Entity\StationStreamer"
 Processing entity "App\Entity\RolePermission"
 Processing entity "App\Entity\StationPlaylist"
 Processing entity "App\Entity\StationMount"
 Processing entity "App\Entity\StationRemote"
 Processing entity "App\Entity\StationWebhook"
 Processing entity "App\Entity\SftpUser"
 Processing entity "App\Entity\ApiKey"
 Processing entity "App\Entity\User"
 Processing entity "App\Entity\UnprocessableMedia"
 Processing entity "App\Entity\PodcastCategory"
 Processing entity "App\Entity\StationMediaCustomField"
 Processing entity "App\Entity\Role"
 Processing entity "App\Entity\StationQueue"
 Processing entity "App\Entity\UserLoginToken"
 Processing entity "App\Entity\Relay"
 Processing entity "App\Entity\MessengerMessage"
 Processing entity "App\Entity\Podcast"
 Processing entity "App\Entity\PodcastMedia"
 Processing entity "App\Entity\StationStreamerBroadcast"
 Processing entity "App\Entity\StationPlaylistMedia"
 Processing entity "App\Entity\StationSchedule"
 Processing entity "App\Entity\CustomField"
 Processing entity "App\Entity\Listener"
 Processing entity "App\Entity\StationPlaylistFolder"
 Processing entity "App\Entity\AuditLog"
 Processing entity "App\Entity\StationRequest"
 Processing entity "App\Entity\PodcastEpisode"

 Proxy classes generated to "/var/azuracast/www_tmp/proxies"

Recargar Datos del Sistema
--------------------------


                                                                                                                        
 [OK] Local cache flushed.                                                                                              
                                                                                                                        


                                                                                                                        
 [OK] All message queues cleared.                                                                                       
                                                                                                                        


Actualizando todas las estaciones
---------------------------------


Restarting all radio stations...
--------------------------------

    0 [▓░░░░░░░░░░░░░░░░░░░░░░░░░░░]


[2021-08-04T19:30:06.595698+00:00] AzuraCast.ERROR: HTTP client GET call to https://central.azuracast.com/ip produced response NULL [] []
[2021-08-04T19:30:06.598027+00:00] AzuraCast.ERROR: Could not fetch remote IP: cURL error 28: Operation timed out after 3000 milliseconds with 0 bytes received (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for https://central.azuracast.com/ip [] []
                                                                                                                        
 [OK] ¡Instalación completa de AzuraCast!                                                                               
                                                                                                                        
      Visite http:// para completar la configuración.                                                                   
                                                                                                                        

azuracast_redis is up-to-date
azuracast_mariadb is up-to-date
azuracast_stations is up-to-date
Creating azuracast_web ... done
Creating nginx_proxy   ... done
Creating nginx_proxy_letsencrypt ... done

**Reply by starapple2** [reply]
So does AzuraCast have to run on its own server and Apache virtual host sites have to be placed elsewhere?

**Comment by curchunflo** [comment]
hi @SlvrEagle23 thank you very much for your answer, now I've uninstalled apache2 and have a new problem, when I go to localhost, I only see nginix homepage, the install's log has some errors, I paste the output here, maybe it helps to know where is the problem. I've run twice the install script, the first one it didnt detect my public ip, the second, it did it, but I want to use it with localhost due I can't configure a static IP in my network.
I can't find where to modify the configuration (there is any file I can edit after installation without get in localhost?)
I am running linux mint

root@xxx:/var/azuracast# ./docker.sh install
Docker is already installed! Continuing...
Docker Compose is already installed and up to date! Continuing...
curl: (22) The requested URL returned error: 404
ERROR: .FileNotFoundError: [Errno 2] No such file or directory: './docker-compose.installer.yml'
ERROR: .FileNotFoundError: [Errno 2] No such file or directory: './docker-compose.installer.yml'
rm: no se puede borrar 'docker-compose.installer.yml': No existe el archivo o el directorio
Pulling mariadb ... done
Pulling redis ... done
Pulling stations ... done
Pulling web ... done
Pulling nginx_proxy ... done
Pulling nginx_proxy_letsencrypt ... done
Creating network "azuracast_backend" with driver "bridge"
Creating network "azuracast_frontend" with driver "bridge"
Creating volume "azuracast_nginx_proxy_vhosts" with default driver
Creating volume "azuracast_db_data" with default driver
Creating volume "azuracast_letsencrypt" with default driver
Creating volume "azuracast_letsencrypt_html" with default driver
Creating volume "azuracast_letsencrypt_acme" with default driver
Creating volume "azuracast_shoutcast2_install" with default driver
Creating volume "azuracast_geolite_install" with default driver
Creating volume "azuracast_sftpgo_data" with default driver
Creating volume "azuracast_station_data" with default driver
Creating volume "azuracast_www_vendor" with default driver
Creating volume "azuracast_tmp_data" with default driver
Creating volume "azuracast_redis_data" with default driver
Creating volume "azuracast_backups" with default driver
Creating azuracast_redis ... done
Creating azuracast_stations ... done
Creating azuracast_mariadb ... done
Creating azuracast_web_run ... done
Starting up AzuraCast services...
Services started up and ready!
AzuraCast Setup
(Environment: production)

Configurar AzuraCast
Bienvenido a AzuraCast. Por favor, espere mientras se configuran algunas dependencias clave de AzuraCast...

Entorno: Production
Método de instalación: Docker
Ejecutando Migraciones de Base de Datos
Generando Clases de Proxy de Base de Datos
Processing entity "App\Entity\Settings"
Processing entity "App\Entity\SongHistory"
Processing entity "App\Entity\Analytics"
Processing entity "App\Entity\StorageLocation"
Processing entity "App\Entity\StationMedia"
Processing entity "App\Entity\Station"
Processing entity "App\Entity\StationStreamer"
Processing entity "App\Entity\RolePermission"
Processing entity "App\Entity\StationPlaylist"
Processing entity "App\Entity\StationMount"
Processing entity "App\Entity\StationRemote"
Processing entity "App\Entity\StationWebhook"
Processing entity "App\Entity\SftpUser"
Processing entity "App\Entity\ApiKey"
Processing entity "App\Entity\User"
Processing entity "App\Entity\UnprocessableMedia"
Processing entity "App\Entity\PodcastCategory"
Processing entity "App\Entity\StationMediaCustomField"
Processing entity "App\Entity\Role"
Processing entity "App\Entity\StationQueue"
Processing entity "App\Entity\UserLoginToken"
Processing entity "App\Entity\Relay"
Processing entity "App\Entity\MessengerMessage"
Processing entity "App\Entity\Podcast"
Processing entity "App\Entity\PodcastMedia"
Processing entity "App\Entity\StationStreamerBroadcast"
Processing entity "App\Entity\StationPlaylistMedia"
Processing entity "App\Entity\StationSchedule"
Processing entity "App\Entity\CustomField"
Processing entity "App\Entity\Listener"
Processing entity "App\Entity\StationPlaylistFolder"
Processing entity "App\Entity\AuditLog"
Processing entity "App\Entity\StationRequest"
Processing entity "App\Entity\PodcastEpisode"

Proxy classes generated to "/var/azuracast/www_tmp/proxies"

Recargar Datos del Sistema
[OK] Local cache flushed.

[OK] All message queues cleared.

Actualizando todas las estaciones
Restarting all radio stations...
0 [▓░░░░░░░░░░░░░░░░░░░░░░░░░░░]
[2021-08-04T19:30:06.595698+00:00] AzuraCast.ERROR: HTTP client GET call to https://central.azuracast.com/ip produced response NULL [] []
[2021-08-04T19:30:06.598027+00:00] AzuraCast.ERROR: Could not fetch remote IP: cURL error 28: Operation timed out after 3000 milliseconds with 0 bytes received (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for https://central.azuracast.com/ip [] []

[OK] ¡Instalación completa de AzuraCast!

  Visite http:// para completar la configuración.                                                                   
azuracast_redis is up-to-date
azuracast_mariadb is up-to-date
azuracast_stations is up-to-date
Creating azuracast_web ... done
Creating nginx_proxy ... done
Creating nginx_proxy_letsencrypt ... done

I don't know wich files are these, it would be helpful to know it so I can fix this errors

curl: (22) The requested URL returned error: 404
ERROR: .FileNotFoundError: [Errno 2] No such file or directory: './docker-compose.installer.yml'
ERROR: .FileNotFoundError: [Errno 2] No such file or directory: './docker-compose.installer.yml'

**Comment by joshopkins787** [comment]
Looks like you might have a networking issue I would guess maybe DNS as the script was not able to get the response from https://central.azuracast.com/ip

I am not sure what OS you are running or if it is hosted server but you might need to reach out to them if it is hosted.

**Reply by Hamza-R00** [reply]
hey man can you please help me too, I am new, and I keep getting this

![unnamed](https://user-images.githubusercontent.com/56293643/162803638-e6d3642b-8335-44cb-b287-de4b5199d546.png)

**Comment by BusterNeece** [comment]
Around the same time as you had this error, we were having trouble with our Central service and were upgrading it, so it's very possible that your installation simply couldn't get the IP address in its initial installation. If you're still experiencing any sort of error as a result of that, you may want to try running the installation again.

**Comment by curchunflo** [comment]
thanks for your answers, on Tuesday after several reinstallations I managed to complete the installation. I read on a forum (sorry I can't say which one) that a lot of people had had this problem and had managed to re-install it until it worked. Surely it is the problem of the server that they commented to me. Greetings and thank you very much for your help

**Comment by curchunflo** [comment]
I only can see a meme, can you please tellme with words please?
________________________________
De: Hamza-R00 ***@***.***>
Enviado: lunes, 11 de abril de 2022 06:16 p. m.
Para: AzuraCast/AzuraCast ***@***.***>
CC: curchunflo ***@***.***>; Mention ***@***.***>
Asunto: Re: [AzuraCast/AzuraCast] [SOLVED] fresh install: can't access AzuraCast (Discussion #4449)


hey man can you please help me too, I am new, and I keep getting this
[1649238581070]<https://user-images.githubusercontent.com/56293643/162803539-af2f25ff-8454-432b-bbe7-be3c238d8007.jpg>
